liferay-workflow/liferay-portal
# LPS-132436 Remove usage and definition of the no-longer-used isShowStructureSelection

Hi @liferay-workflow! 👋 

This is a PR that removes `isShowStructureSelection` which is apparently dead code. I wanted to migrate the `openDDMPortlet` but @rafaprax warned me it's probably dead code.

I've removed the usages and definitions, and tested and everything checks out fine.

# 🧪 Testing
1. Make sure you have access to Kaleo Forms Admin
2. Start creating a new Process
3. On Forms step create a new Form and assign it to the Task
